src/App.tsx

```diff
@@ -1,17 +1,133 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
+import cn from 'classnames';
 
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { PreparedTodo } from './app.typedef';
+
+const preparedTodos: PreparedTodo[] = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => {
+  const [todosList, setTodosList] = useState([...preparedTodos]);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [titleInputError, setTitleInputError] = useState(false);
+  const [userInputError, setUserInputError] = useState(false);
+
+  const addtodo = (title: string, userId: number) => {
+    const newTodo: PreparedTodo = {
+      userId,
+      id: Math.max(...todosList.map((todo) => todo.id)) + 1,
+      title,
+      completed: false,
+      user: users.find(user => userId === user.id) || null,
```

Comment:
maybe if the user does not exist we should not add a todo? otherwise who will do it?